packages/block-library/src/image/interactivity.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Internal dependencies
+ */
+import { store } from '../utils/interactivity';
+
+const raf = window.requestAnimationFrame;
+// Until useSignalEffects is fixed: https://github.com/preactjs/signals/issues/228
+const tick = () => new Promise( ( r ) => raf( () => raf( r ) ) );
```

Comment:
Now that [this pull request](https://github.com/WordPress/block-interactivity-experiments/pull/226) has been merged, this workaround shouldn't be necessary. Could you please try merging the `add-interactivity-runtime` branch and check if everything works as expected without using `await tick()`?